src/sif/autocost.cc

```diff
@@ -697,7 +684,7 @@ void ParseAutoCostOptions(const rapidjson::Document& doc,
   const auto& json = rapidjson::get_child(doc, costing_options_key.c_str(), dummy);
 
   ParseBaseCostOptions(json, c, kBaseCostOptsConfig);
-  JSON_PBF_DEFAULT(co, kDefaultAutoType, json, "/type", transport_type);
+  // JSON_PBF_DEFAULT(co, kDefaultAutoType, json, "/type", transport_type);
```

Comment:
I left this here, it's very likely we'll use it at some point. I might have case for RV routing soon-ish, though not sure if that wouldn't be better done inside truck costing.